modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -28,6 +32,12 @@ public function get_icon() {
 
 	protected function render() {
 		$settings = $this->get_atomic_settings();
+		$is_link_enabled = ! empty( $settings['link']['enabled'] ) && ! empty( $settings['link']['href'] );
```

Comment:
Following my comment on the right prop resolver to use, instead of checking the `enabled` flag, we would probably check that `href` is set (as there shouldn't really be an `enabled` attribute there)